### PR TITLE
layout design user

### PR DESCRIPTION
**Pull request by adi061001**
First ticket  done .

### CODE REVIEW

File: client/src/Layout/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+const FooterBox = styled(Box)(({ theme }) => ({
+  "background-color": " black",
+  padding: "30px 0px 30px 0px",
+}))
+
+const FooterInput = styled(Box)(({ theme }) => ({
+  border: "1px solid white",
+  width: "210px",
+  height: " 46px",
+  " margin-top": "16px",
```
Comment (tw-vineet) [inline client/src/Layout/Footer.tsx]: use camel  case

File: client/src/Layout/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                  <InstagramIcon />
+                  <TwitterIcon />
+                </Box>
+              </Box>
+            </Grid2>
+          </Grid2>
+        </ConatinerFooter>
+        <StyledDivider />
+        <Box textAlign={"center"}>
+          <Typography variant="subtitle2" sx={{ color: "gray", mt: 2 }}>
```
Comment (tw-vineet) [inline client/src/Layout/Footer.tsx]: you are repeating same mistake again and again  please  do not use inline style

File: client/src/Dto/dataType.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  userName: String
+  userEmail: String
+  userPhone: String
+  userPassword: String
+}
+
+export interface IFooterData {
+  Support: string[]
+  Account: string[]
+  "Quick Links": string[]
```
Comment (tw-vineet) [inline client/src/Dto/dataType.ts:15]: why are you using these type of syntax , please use camel case

File: client/src/Layout/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+                    variant="filled"
+                    styleProps={{
+                      "& .MuiInputLabel-filled": {
+                        color: "white",
+                      },
+                      "& .MuiFilledInput-underline:after": {
+                        border: 0,
+                      },
+                    }}
+                  ></Input>
```
Comment (tw-vineet) [inline client/src/Layout/Footer.tsx]: please use self closing tag

File: client/src/Layout/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+              </Box>
+            </Grid2>
+
+            {Object.entries(footerData).map((value: [string, []], key: number) => {
+
+              return (
+                <>
+                  <Grid2 size={2}>
+                    <Box>
+                      <Footertext variant="h6">{value[0]}</Footertext>
```
Comment (tw-vineet) [inline client/src/Layout/Footer.tsx]: avoid using indexing

File: client/src/logo.svg
Comment (tw-vineet) [inline client/src/logo.svg:1]: avoid unwanted image

File: client/src/Layout/Footer.tsx (hunk 1 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx]: ohk

File: client/src/Layout/Footer.tsx (hunk 4 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx]: ohk

File: client/src/Layout/Footer.tsx (hunk 5 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx]: ohk

File: client/src/logo.svg
Comment (adi061001) [reply client/src/logo.svg:1]: ohk

File: client/src/Layout/Footer.tsx (hunk 2 above)
Comment (adi061001) [reply client/src/Layout/Footer.tsx]: I am really sorry for that

File: client/src/Dto/dataType.ts (hunk 3 above)
Comment (shubh123thoughtwin) [reply client/src/Dto/dataType.ts:15]: I have told him to do, that's how we define interface.

**Pull request merged**